server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/StdJobScheduler.java

```diff
@@ -160,6 +165,7 @@ private void tryCanceling(Long jobId) throws JobException {
 
     private void initDaemonJob() {
         initCheckRunningJob();
+        initPullTaskResultJob();
```

Comment:
should we disable the TaskMonitor when the TaskMonitorMode is 'PULL'?

Reply:
Yes, actually it is disabled while monitor mode is push